packages/firebase_snippets_app/database.rules.json

```diff
@@ -1,6 +1,14 @@
 {
   "rules": {
-    ".read": "auth != null",
-    ".write": "auth != null"
+    ".read": "false",
+    ".write": "false"
+  },
+  "posts": {
```

Comment:
Ooh, much better 👍

The `posts` and `users` nodes have to be under the `rules` node though, so can you move them there.